bigquery/spec/bigquery_sample_spec.rb

```diff
@@ -28,35 +28,40 @@
     @bucket     = @storage.bucket ENV["GOOGLE_CLOUD_STORAGE_BUCKET"]
     @tempfiles  = []
 
-    # Examples assume that newly created test_dataset and test_table exist
-    delete_test_dataset!
+    file_time     = Time.now.to_i
```

Comment:
What does "file time" mean?

What about calling this `timestamp` or `current_time`?